influence_benchmark/config/experiment_config.py

```diff
@@ -143,7 +143,8 @@ class LocalTrainingConfig(BaseExperimentConfig):
     report_to: str
     optim: str
     max_length: int
-    lr_scheduler_type: str
+    lr_scheduler_type: str  # (Within each iteration)
+    across_iter_lr_decay_rate: float  # (Across iterations) E.g. if 0.5, LR will be halved after every iteration
```

Comment:
Probably not be worth changing but it might be a bit confusing to call it decay rate? if i understand it correctly 1 means no decay so this is like a not decay rate. Also your example is the only possible one where it is not clear which it means since both a decay rate of 0.5 and a not decay rate of 0.5 are the same

Reply:
loool fair